tests/integration/pilot/analysis/analysis_test.go

```diff
@@ -30,12 +30,42 @@ import (
 	"istio.io/api/meta/v1alpha1"
 	"istio.io/istio/pkg/config/analysis/msg"
 	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/istioctl"
 	"istio.io/istio/pkg/test/framework/components/namespace"
 	"istio.io/istio/pkg/test/framework/features"
 	"istio.io/istio/pkg/test/framework/label"
 	"istio.io/istio/pkg/test/util/retry"
 )
 
+func TestWait(t *testing.T) {
```

Comment:
I'm wondering why this istioctl test is being moved out of `istioctl_test.go`?

Reply:
Yes， because that feature flag was not enabled there

Reply:
I see, thanks